pkg/ansible/controller/reconcile.go

```diff
@@ -72,7 +72,7 @@ func (r *AnsibleOperatorReconciler) Reconcile(request reconcile.Request) (reconc
 		return reconcile.Result{}, err
 	}
 
-	ident := strconv.Itoa(rand.Int())
+	ident := strconv.Itoa(rand.Int()) // #nosec - suppressing "G404: Use of weak random number generator (math/rand instead of crypto/rand)"
```

Comment:
Why not switch to use `crypto/rand`?

Reply:
Before discussing any change do you think that we should move forward with the gosec check or could not use it at all?